src/scripts/farming/FarmController.ts

```diff
@@ -80,25 +80,42 @@ class FarmController {
         return MulchType[plot.mulch];
     }
 
-    public static plotClick(index: number) {
+    public static plotClick(index: number, event: MouseEvent) {
         const plot: Plot = App.game.farming.plotList[index];
+
+        if (event.shiftKey) {
+            this.handleShiftClickActions(plot, index);
+        } else {
+            this.handleClickActions(plot, index);
+        }
+    }
+
+    private static handleShiftClickActions(plot: Plot, index: number) {
```

Comment:
I think it would be better if the method name described what it did, instead of when it should be used.